### PR TITLE
Update old code

### PR DESCRIPTION
**Pull request by XiosBombay**

### CODE REVIEW

File: fxmanifest.lua [hunk 1]
```diff
@@ -1,7 +1,11 @@
 fx_version 'cerulean'
-game 'gta5'
+
+game { 'gta5' }
 
 author 'Deli'
+
```
Comment (tabarra) [inline fxmanifest.lua:6]: Is your code doing social distancing?

File: drift.lua [hunk 2]
```diff
@@ -1,36 +1,22 @@
...
-		end
-
-		if driftmode then
-			if IsPedInAnyVehicle(GetPed(), false) then
-				CarSpeed = GetEntitySpeed(GetCar()) * speed
-				if GetPedInVehicleSeat(GetCar(), -1) == GetPed() then
-					if CarSpeed <= drift_speed_limit then  
-						if IsControlPressed(1, 21) then
-							SetVehicleReduceGrip(GetCar(), true)
+		Citizen.Wait(970)
```
Comment (tabarra) [inline drift.lua:10]: too arbitrary, 250 is responsive enough.

File: drift.lua [hunk 3]
```diff
@@ -39,7 +25,14 @@ Citizen.CreateThread(function()
 	end
 end)
 
+RegisterCommand('driftmode', function()
+	if not driftMode then
```
Comment (tabarra) [inline drift.lua:29]: Why?
```
driftmode = not driftmode
Set...(..., driftmode)
```

**Comment by HypeLevels** [comment]
sooo do I merge it or not!?!?!? @XiosBombay mind taking a look to what @tabarra said?

**Comment by XiosBombay** [comment]
> sooo do I merge it or not!?!?!? @XiosBombay mind taking a look to what @tabarra said?

Sorry. I was sick. I'll fix.

File: drift.lua (hunk 3 above)
Comment (XiosBombay) [reply drift.lua:29]: I don't see a benefit in using driftmode = not driftmode

**Comment by HypeLevels** [comment]
merging since @tabarra didnt say anything against these 2 commits

**Pull request merged**